blueprint/src/chapters/stochastic_integral.tex

```diff
@@ -9,6 +10,113 @@ \section{Total variation and Lebesgue-Stieltjes integral}
+  By Jensen
+  $\phi(X_t) = \phi\left( \mathbb{E}[X_T\ |\ \mathcal{F}_t] \right)\leq \mathbb{E}[\phi(X_T)\ |\ \mathcal{F}_t]$.
+\end{proof}
+
+\begin{lemma}\label{lem:convex_of_submg_is_submg}
+  Let $X:T\times\Omega\rightarrow \mathbb{R}^d$ a sub-martingale.
+  Let $\phi:\mathbb{R^d}\rightarrow \mathbb{R}$ convex increasing such that
+  $\phi(X_t)\in L^1(\Omega)$ for every $t\in T$. Then $\phi(X)$ is a sub-martingale.
+\end{lemma}
+\begin{proof}
+  By Jensen and the fact that $\phi$ is increasing
+  $\phi(X_t) \leq \phi\left( \mathbb{E}[X_T\ |\ \mathcal{F}_t] \right)\leq \mathbb{E}[\phi(X_T)\ |\ \mathcal{F}_t]$.
+\end{proof}
+
+\begin{lemma}[Doob Inequality for countable]\label{lem:doob_countable}
+  Let $X:I\times\Omega\rightarrow \mathbb{R}$ be a non-negative sub-martingale.  Let $I$ be countable.
+  For every $M\in I,\lambda > 0$ and $p>1$ we have
+  $$
+  P\left( \sup_{i\in I, i\leq M}X_i\geq\lambda \right)\leq \frac{\mathbb{E}[X_M]}{\lambda}.
+  $$
+\end{lemma}
+\begin{proof}
+  \uses{lem:convex_of_mg_is_submg}
+  See 8.1.1 Pascucci.
+\end{proof}
+
+\begin{lemma}[Doob Inequality Corollary for countable]\label{lem:doob_countable_cor}
+  Let $X:I\times\Omega\rightarrow \mathbb{R}$ be a sub-martingale. Let $I$ be countable.
+  For every $M\in I,\lambda > 0$ and $p>1$ we have
+  $$
+  \mathbb{E}\left[ \sup_{i\in I, i\leq M}X_i^p \right]\leq \left(\frac{p}{p-1}\right)^p\mathbb{E}[X_M^p].
+  $$
+\end{lemma}
+\begin{proof}
+  \uses{lem:convex_of_mg_is_submg,lem:doob_countable}
+  8.1.1 Pascucci.
+\end{proof}
+
+\begin{theorem}[Doob Inequality]\label{thm:doob_ineq}
+  Let $X:\mathbb{R}\times\Omega\rightarrow \mathbb{R}$ be a right-continuous non-negative sub-martingale.
+  For every $T, \lambda>0$ and $p>1$ we have
+  $$
+  P\left( \sup_{t\in[0,T]}X_t\geq\lambda \right)\leq \frac{\mathbb{E}[X_T]}{\lambda}.
+  $$
+\end{theorem}
+\begin{proof}
+  \uses{lem:doob_countable}
+  8.1.2 Pascucci.
+\end{proof}
+
+\begin{theorem}[Doob's Lp inequality]\label{thm:doob_lp}
+  Let $X:\mathbb{R}\times\Omega\rightarrow \mathbb{R}$ be a right-continuous non-negative sub-martingale.
+  For every $T, \lambda>0$ and $p>1$ we have
+  $$
+  \mathbb{E}\left[ \sup_{t\in[0,T]}X_t^p \right]\leq \left(\frac{p}{p-1}\right)^p\mathbb{E}[X_T^p].
+  $$
+\end{theorem}
+\begin{proof}
+  \uses{lem:doob_countable_cor}
+  8.1.2 Pascucci.
+\end{proof}
+
+\begin{lemma}[Stopped Martingale]\label{lem:stop_of_mg_is_mg}
+  Let $X:\mathbb{R}\times\Omega\rightarrow \mathbb{R}$ be a cadlag martingale and $\tau_0$ a stopping time. Then $(X_{t\wedge\tau_0})_{t\geq 0}$ is a martingale.
+\end{lemma}
+
+\begin{lemma}[Doob Inequality for stopping times]\label{lem:doob_ineq_stop}
+  Let $X:\mathbb{R}\times\Omega\rightarrow \mathbb{R}$ be a right-continuous non-negative sub-martingale.
+  For every $\lambda>0$ and $p>1$ and $\tau$ stopping time a.s. bounded by $T>0$, we have
+  $$
+  P\left( \sup_{t\in[0,\tau]}|X_t|\geq\lambda \right)\leq \frac{\mathbb{E}[|X_\tau|]}{\lambda}.
+  $$
```

Comment:
Can you write that lemma for X taking value in a normed space? The inequality will then be on the norm, which is a nonnegative submartingale (that fact needs a lemma).